src/client/Health/NodeView.jsx

```diff
@@ -113,8 +438,10 @@ const NodeView = ({ status, onRestart, onShutdown }) => {
       isHidden: !dataSource.enabled,
     }
   )) ?? []
+  const styleEngineSettings = { background: 'rgb(152,151,151)', border: 'none', position: 'absolute', bottom: '-5px', right: '0px' }
```

Comment:
Can't you use a css class instead of declaring the style inside the react component ?